src/legacy/ui/public/embeddable/context_menu_actions/context_menu_action.ts

```diff
@@ -69,11 +74,6 @@ interface ContextMenuLinkOptions extends ContextMenuActionOptions {
 interface ContextMenuActionsConfig {
   id: string;
 
-  /**
-   * Display name of the action in the menu
-   */
-  displayName: string;
```

Comment:
I'm the _slightest_ bit worried that we might need to keep this around for bw compatibility sake in case there's plugins out there that use this in 7.x range. Not sure if this is actually something to be worried about, but wanted to raise it just in case. Would it be terribly hard to make it backwards compatible (use the hard-coded title if not getter is provided?)

Reply:
I could keep both displayName and getDisplayName and then check for one before the other. This will break the typescript interface checking because there is no way to verify that either displayName or getDisplayName exists.

There is lots of precedent for breaking changes in Kibana's interfaces between minors. So I see no problem with this breaking change in a minor.

With that said, I have no preference and will defer to @elastic/kibana-app. Which is preferred?

Reply:
I have a feeling BWC won't be a huge concern on this, but would be good to make sure it aligns with the current embeddables work that @stacey-gammon is doing.